Recrutify/Recrutify.Host/Startup.cs

```diff
@@ -54,6 +57,16 @@ public void ConfigureServices(IServiceCollection services)
                     .AllowAnyMethod());
             });
 
+            services.AddSingleton<IAuthorizationHandler, RolesPolicyHandler>();
```

Comment:
move this registrations lower in startup, how it was: below swagger and indentity